reporting/reporting_calm_transformer/src/reporting_calm_transformer.py

```diff
@@ -0,0 +1,33 @@
+"""
+Basic transformer, which cleans up the static calm data before sending it off
+to an elasticsearch indesx. The raw data can be downloaded by running:
+
+    python monitoring/scripts/download_oai_harvest.py
+
+from the root of this repo. The `path_to_calm_json` parameter below should be
+set to the relative path to that file from this directory. The `es_username`,
```

Comment:
I'm not sure I understand what this parameter should be. It's probably also not necessary – you can work it out.

```python
# Path to the root of the Git repo
subprocess.check_call([
    "git", "rev-parse", "--show-toplevel"]).strip().decode("utf8")

# Path to the current file
os.path.abspath(__file__)
```